doc/045_working_with_repos.rst

```diff
@@ -87,3 +87,29 @@ yield the same error:
     Load indexes
     ciphertext verification failed
 
+By default, ``check`` command does not check data packs integrity. 
```

Comment:
replace "data packs integrity" with "that files stored in the repository are unmodified". This is end-user documentation, being less technical is a feature :)

Reply:
Maybe too fine a point here, but I think most users will assume that "files stored in the repository" mean "_user_ files backed up to the repository", which ``check`` does not actually check (see #1620). I purposely used very technical "data packs" to avoid this confusion. I now changed the text to use less technical "repository data files", which I hope will be less likely confused with "user files stored in the repository". Let me know if you prefer different wording.